.github/workflows/test-apt-packages.yml

```diff
@@ -0,0 +1,82 @@
+name: Run tests on APT packages
```

Comment:
[OPTIONAL]

I think a more accurate name (basic smoke / startup test?) might be better if we are not planning to extend it with more tests.

Reply:
Thanks for the review note.
Fixed.